File: app/src/main/java/com/motrixi/datacollection/utils/MessageUtil.java
package com.motrixi.datacollection.utils;

import org.json.JSONException;
import org.json.JSONObject;

/**
 * author : Jason
 * date   : 2020/12/8 11:06 AM
 * desc   :
 */
public class MessageUtil {

    public static String logMessage(int type, boolean success, String msg) {

        try {
            JSONObject jsonObject = new JSONObject();
            jsonObject.put("type", type);
            jsonObject.put("success", success);
            jsonObject.put("info", msg);

            return jsonObject.toString();
        } catch (JSONException e) {
            e.printStackTrace();
            return "";
        }
    }

    public static String uploadLog(String msg) {

        try {
            JSONObject jsonObject =  new JSONObject();
            jsonObject.put("info", msg);

//            return jsonObject.toString()
            return msg;
        } catch (JSONException e) {
            e.printStackTrace();
            return "";
        }
    }
}

File: app/src/main/java/com/motrixi/datacollection/utils/MessageUtil.kt
package com.motrixi.datacollection.utils

import org.json.JSONObject

/**
 * author : Jason
 *  date   : 2020/11/18 12:53 PM
 *  desc   :
 */
object MessageUtil {

    fun logMessage(type: Int, success: Boolean, msg: String): String {

        try {
            var jsonObject = JSONObject()
            jsonObject.put("type", type)
            jsonObject.put("success", success)
            jsonObject.put("info", msg)

            return jsonObject.toString()
        } catch (e: Exception) {
            return ""
        }
    }

    fun uploadLog(msg: String): String {

        try {
            var jsonObject = JSONObject()
            jsonObject.put("info", msg)

//            return jsonObject.toString()
            return msg
        } catch (e: Exception) {
            return ""
        }
    }
}